src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -133,8 +136,57 @@ def consent_request_verify(
     return _prepare_consent_preferences(db, provided_identity)
 
 
-@router.patch(
+@router.post(
     CONSENT_REQUEST_PREFERENCES,
+    dependencies=[Security(verify_oauth_client, scopes=[CONSENT_READ])],
+    status_code=HTTP_200_OK,
+    response_model=ConsentPreferences,
+)
+def get_consent_preferences(
```

Comment:
Again, haven't been in any discussions around this feature. so I'm confused about the purpose of this endpoint versus "consent_request_verify".  Is `get_consent_preferences` to be used by fidesops and `consent_request_verify` to be hit by the user themselves?